glean-core/ffi/src/ping_type.rs

```diff
@@ -2,18 +2,19 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#![allow(clippy::redundant_closure)]
+
 use ffi_support::{ConcurrentHandleMap, FfiStr};
-use lazy_static::lazy_static;
+use once_cell::sync::Lazy;
 
 use glean_core::metrics::PingType;
 
 use crate::ffi_string_ext::FallibleToString;
 use crate::handlemap_ext::HandleMapExtension;
 use crate::{from_raw_string_array, with_glean_value, with_glean_value_mut, RawStringArray};
 
-lazy_static! {
-    pub(crate) static ref PING_TYPES: ConcurrentHandleMap<PingType> = ConcurrentHandleMap::new();
-}
+pub(crate) static PING_TYPES: Lazy<ConcurrentHandleMap<PingType>> =
```

Comment:
Same here for avoiding hitting that lint.
It's not in a macro, so importing the type and using the short name is better.

Reply:
Thanks, I knew there had to be an easier way :)